doc/usage.rst

```diff
@@ -64,191 +63,41 @@ Config file:
-
-.. code:: python
-
-    >>> X, names = dataset.get_data(return_attribute_names=True)
-    >>> print(names)
-    ['Wifes_age', 'Wifes_education', 'Husbands_education', 'Number_of_children_ever_born', 'Wifes_religion', 'Wifes_now_working%3F', 'Husbands_occupation', 'Standard-of-living_index', 'Media_exposure', 'Contraceptive_method_used']
-
-Most times, having a single data matrix :python:`X` is not enough. Two 
-useful arguments are :python:`target` and
-:python:`return_categorical_indicator`. :python:`target` makes
-:meth:`get_data()` return :python:`X` and :python:`y`
-seperate; :python:`return_categorical_indicator` makes
-:meth:`get_data()` return a boolean array which indicate
-which attributes are categorical (and should be one hot encoded if necessary.)
-
-.. code:: python
-
-    >>> X, y, categorical = dataset.get_data(
-    ... target=dataset.default_target_attribute,
-    ... return_categorical_indicator=True)
-    >>> print(X.shape, y.shape)
-    (1473, 9) (1473,)
-    >>> print(categorical)
-    [False, True, True, False, True, True, True, True, True]
 
-In case you are working with `scikit-learn
-<http://scikit-learn>`_, you can use this data right away:
+~~~~~~~~~~~~
+Key concepts
+~~~~~~~~~~~~
 
-.. code:: python
-
-    >>> from sklearn import preprocessing, ensemble
-    >>> enc = preprocessing.OneHotEncoder(categorical_features=categorical)
-    >>> print(enc)
-    OneHotEncoder(categorical_features=[False, True, True, False, True, True, True, True, True],
-           dtype=<class 'numpy.float64'>, handle_unknown='error',
-           n_values='auto', sparse=True)
-    >>> X = enc.fit_transform(X).todense()
-    >>> clf = ensemble.RandomForestClassifier()
-    >>> clf.fit(X, y)
-    RandomForestClassifier(bootstrap=True, class_weight=None, criterion='gini',
-                max_depth=None, max_features='auto', max_leaf_nodes=None,
-                min_impurity_split=1e-07, min_samples_leaf=1,
-                min_samples_split=2, min_weight_fraction_leaf=0.0,
-                n_estimators=10, n_jobs=1, oob_score=False, random_state=None,
-                verbose=0, warm_start=False)
-
-When you have to retrieve several datasets, you can use the convenience function
-:meth:`openml.datasets.get_datasets()`, which downloads all datasets given by
-a list of IDs:
-
-    >>> ids = [12, 14, 16, 18, 20, 22]
-    >>> datasets = openml.datasets.get_datasets(ids)
-    >>> print(datasets[0].name)
-    mfeat-factors
+OpenML contains several key concepts which it needs to make machine learning
+research shareable. A machine learning experiment consists of several runs,
+which describe the performance of an algorithm (called a flow in OpenML) on a
+task. Task is the combination of a dataset, a split and an evaluation metric. In
+this user guide we will go through listing and exploring existing tasks to
+actually running machine learning algorithms on them. In a further user guide
```

Comment:
Maybe say "run is flow + setup + task and produces metric and predictions"? Right now you don't explain "run" right? Maybe make the key concepts bold.